sources/packages/api/src-sql/DisbursementReceipts/Create-disbursement-receipts-table.sql

```diff
@@ -0,0 +1,64 @@
+--Create table sims.disbursement_receipts.
+CREATE TABLE IF NOT EXISTS sims.disbursement_receipts(
+    id SERIAL PRIMARY KEY,
+    batch_run_date DATE NOT NULL,
+    student_sin CHAR(9) NOT NULL,
+    disbursement_schedule_id INT NOT NULL REFERENCES sims.disbursement_schedules(id),
+    funding_type CHAR(2) NOT NULL,
+    total_entitled_disbursed_amount NUMERIC(5) NOT NULL,
+    total_disbursed_amount NUMERIC(5) NOT NULL,
+    disburse_date DATE NOT NULL,
+    disburse_amount_student NUMERIC(5) NOT NULL,
+    disburse_amount_institution NUMERIC(5) NOT NULL,
+    date_signed_institution DATE NOT NULL,
+    institution_code CHAR(4) NOT NULL,
+    disburse_method_student CHAR(1) NOT NULL,
```

Comment:
Is it better to save here the white space or maybe allow null for the case "<space> disbursement only to institution"?